src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1220,49 +1223,275 @@ static bool addTriggeredSubscriptions
       continue;
     }
 
     AttributeList aList;
 
-    aList.fill(sP->attributes);
+    aList.fill(cSubP->attributes);
 
     // Throttling
-    if ((sP->throttling != -1) && (sP->lastNotificationTime != -1))
+    if ((cSubP->throttling != -1) && (cSubP->lastNotificationTime != 0))
     {
-      if ((now - sP->lastNotificationTime) < sP->throttling)
+      if ((now - cSubP->lastNotificationTime) < cSubP->throttling)
       {
+        LM_T(LmtMongoSubCache, ("subscription '%s' ignored due to throttling", cSubP->subscriptionId));
         continue;
       }
+      else
+      {
+        cSubP->pendingNotifications += 1;
+
+        LM_T(LmtMongoSubCache, ("subscription '%s' NOT ignored due to throttling (T: %lu, LNT: %lu, NOW: %lu, NOW-LNT: %lu, T: %lu)",
+                                cSubP->subscriptionId,
+                                cSubP->throttling,
+                                cSubP->lastNotificationTime,
+                                now,
+                                now - cSubP->lastNotificationTime,
+                                cSubP->throttling));
+      }
+    }
+    else
+    {
+      cSubP->pendingNotifications += 1;
+
+      LM_T(LmtMongoSubCache, ("subscription '%s' NOT ignored due to throttling II (T: %lu, LNT: %lu, NOW: %lu, NOW-LNT: %lu, T: %lu)",
+                              cSubP->subscriptionId,
+                              cSubP->throttling,
+                              cSubP->lastNotificationTime,
+                              now,
+                              now - cSubP->lastNotificationTime,
+                              cSubP->throttling));
     }
 
-    TriggeredSubscription* sub = new TriggeredSubscription((long long) sP->throttling,
-                                                           (long long) sP->lastNotificationTime,
-                                                           sP->format,
-                                                           sP->reference.get(),
+    TriggeredSubscription* sub = new TriggeredSubscription((long long) cSubP->throttling,
+                                                           (long long) cSubP->lastNotificationTime,
+                                                           cSubP->notifyFormat,
+                                                           cSubP->reference,
                                                            aList,
-                                                           sP);
-    subs.insert(std::pair<string, TriggeredSubscription*>(sP->subscriptionId, sub));
+                                                           cSubP->subscriptionId,
+                                                           cSubP->tenant);
+    subs.insert(std::pair<string, TriggeredSubscription*>(cSubP->subscriptionId, sub));
   }
 
   return true;
 }
 
 
+
+/* ****************************************************************************
+*
+* addTriggeredSubscriptions_noCache
+*
+* Recovered almost verbatim from release 0.23.0
```

Comment:
FIXME inserted. I can't find the issue though. Did you perhaps mean to create a new issue about it?